### PR TITLE
修改和添加dubbo社区内容

### PR DESCRIPTION
**Pull request by caojiele**
## What is the purpose of the change

   Add English version of Shenzhen Meetup and Chengdu Meetup summary. Before I could put the modified submit again, but it doesn't matter, this submissions is January 18 commits version.       
   (ps: issue 👉👉👉#237)

### CODE REVIEW

**Comment by lovepoem** [comment]
please  resolve conflicts

File: docs/zh-cn/user/references/registry/zookeeper.md [hunk 1]
```diff
@@ -135,7 +135,8 @@ Zookeeper 集群配置：
 
 ## zookeeper 安装
 
-安装方式参见: [Zookeeper安装手册](../../../admin/install/zookeeper.md)，只需搭一个原生的 Zookeeper 服务器，并将 [Quick Start](../../preface/usage.md) 中 Provider 和 Consumer 里的 `conf/dubbo.properties` 中的 `dubbo.registry.addrss` 的值改为 `zookeeper://127.0.0.1:2181` 即可使用。
+安装方式参见: [Zookeeper安装手册](http://dubbo.apache.org/zh-cn/docs/admin/install/zookeeper.html)，只需搭一个原生的 Zookeeper 服务器，并将 [Quick Start](../../quick-start.md) 中 Provider 和 Consumer 里的 `conf/dubbo.properties` 中的 `dubbo.registry.addrss` 的值改为 `zookeeper://127.0.0.1:2181` 即可使用。
```
Comment (ralf0131) [inline docs/zh-cn/user/references/registry/zookeeper.md]: The link looks good to me, why do you want to change it?

File: en-us/blog/dubbo-meetup-chengdu.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```
Comment (ralf0131) [inline en-us/blog/dubbo-meetup-chengdu.html]: Please do not update html files. Please refer to the README before sending a pull request.

File: docs/zh-cn/user/references/registry/zookeeper.md (hunk 1 above)
Comment (caojiele) [reply docs/zh-cn/user/references/registry/zookeeper.md]: Ok，actually two links are the same, choose which one is the same.

File: en-us/blog/dubbo-meetup-chengdu.html (hunk 2 above)
Comment (caojiele) [reply en-us/blog/dubbo-meetup-chengdu.html]: Is not a issue #237  didn't solve?

File: docs/zh-cn/user/references/registry/zookeeper.md (hunk 1 above)
Comment (ralf0131) [reply docs/zh-cn/user/references/registry/zookeeper.md]: I'd prefer not change it since it is not broken.

Comment (caojiele) [reply docs/zh-cn/user/references/registry/zookeeper.md]: ok

File: docs/zh-cn/dev/principals/dummy.md [hunk 3]
```diff
@@ -105,9 +105,9 @@ public final class Version {
             if (version == null || version.length() == 0) {  
                 // 如果MANIFEST.MF规范中没有版本号，基于jar包名获取版本号  
                 String file = cls.getProtectionDomain().getCodeSource().getLocation().getFile();  
-                if (file != null && file.length() > 0 && file.endsWith(".jar")) {  
+                if (file != null &amp;&amp; file.length() &gt; 0 &amp;&amp; file.endsWith(".jar")) {
```
Comment (ralf0131) [inline docs/zh-cn/dev/principals/dummy.md]: You shouldn't change here. What you should change is Line 37 and Line 43

Comment (caojiele) [reply docs/zh-cn/dev/principals/dummy.md]: OK，I have already corrected.

**Comment by ralf0131** [comment]
Sorry, I can't merge your pull request. Please  remove all the change to html, json, etc. We only need to update .md files.

**Comment by caojiele** [comment]
@ralf0131 Then I turn off the request.

**Pull request closed without merging**